Scripts/RemediationScripts/Helper.ps1

```diff
@@ -0,0 +1,122 @@
+Set-StrictMode -Version Latest
+
+class ResourceResolver
+{
+	[string[]] $ExcludeResourceNames=@();
+	[PSObject] $ExcludedResources=@();
+	[string[]] $ExcludeResourceGroupNames=@();
+	[string[]] $printMessage=@();
```

Comment:
Rename this to something like messages = @()

Reply:
Done